pkg/virtctl/config_ssh/config_ssh.go

```diff
@@ -0,0 +1,221 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+
+package config_ssh
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/tools/clientcmd"
+
+	k6sv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+
+	"github.com/kevinburke/ssh_config"
+)
+
+const (
+	CONFIG_SSH_COMMAND = "config-ssh"
+	KubeVirtEOLComment = "Generated by KubeVirt"
```

Comment:
This should probably contain some way to identify the context so it's possible to remove only the entries created for one namespace.
And then maybe have `--remove` and `--remove-all`?

Reply:
This may make sense. I left it for now with `--remove`, to keep it simple with a simple remove option which does not need any kubevirt configuration.